chartfx-chart/src/main/resources/de/gsi/chart/ui/titled-border.css

```diff
@@ -1,19 +1,18 @@
-
 .bordered-titled-title {
-  -fx-padding: 0 2 0 2;
-  -fx-font-size: 14px;
-  -fx-background-color: -fx-background;
-  -fx-translate-y: -9;
-  -fx-translate-X: +4;
+    -fx-padding: 0 2 0 2;
+    -fx-font-size: 14px;
+    -fx-background-color: -fx-background;
+    -fx-translate-y: -9;
+    -fx-translate-X: +4;
 }
 
 .bordered-titled-border {
-  -fx-content-display: top;
-  -fx-border-insets: 8 5 5 5;
-  -fx-border-color: black;
-  -fx-border-width: 1;
+    -fx-content-display: top;
+    -fx-border-insets: 8 5 5 5;
+    -fx-border-color: black;
+    -fx-border-width: 1;
 }
 
 .bordered-titled-content {
-  -fx-padding: 10 5 5 5;
-}
+    -fx-padding: 10 5 5 5;
+}
```

Comment:
Codacy found an issue: [Unexpected missing end-of-source newline (no-missing-end-of-source-newline)](https://app.codacy.com/manual/GSI/chart-fx/pullRequest?prid=5780860)